server-go/api/api_response.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Grafeas API
+ *
+ * An API to insert and retrieve annotations on cloud artifacts.
+ *
+ * OpenAPI spec version: 0.1
+ *
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+
+package api
+
+import (
+	"net/http"
+)
+
+type APIResponse struct {
```

Comment:
We don't need the APIResponse/APIClient/Configuration here, since what we really want is a model for the server.

Can you remove these and add them to the .swagger-codegen-ingore? That should also remove the resty dep which is causing the build to fail.

Reply:
I don't see that it is ignored in master. :-\

Reply:
also, what did you use to do the vendor.json? not glide, godep or dep

Reply:
govendor

I _should_ have ignored in master. I just deleted the files initially, I planned to come back to it. You got there first :)

Reply:
oh! that makes sense. The `.swagger-codegen-ignore` seems to ignore patterns of file paths, not of structs to _not_ be rendered. I will try my hand at this. And investigate govendor for future use. Shame it leaks your local paths :-\